keystorage/yubihsm2/src/main/java/tech/pegasys/signers/yubihsm/OpaqueDataFormat.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.yubihsm;
+
+public enum OpaqueDataFormat {
+  ASCII,
+  HEX
```

Comment:
Wondering if we even want to differentiate this at this level. What is stored in the opaque data is Bytes (I assume) - if you as the client want to use it as ASCII, go for it - if its base64encoded, also fine .... i.e. the extraction maybe shouldn't be responsible for the conversion (that's application specific).